---
geometry_msgs/Quaternion init_orientation
std_msgs/Float64[3] gyro_bias
